### ISSUE TITLE
Population simulation: "Export population to csv" should export aging data as well

### ISSUE THREAD

**Issue by Yuri05** [Open-Systems-Pharmacology/PK-Sim#1405]
At the moment only population.csv is exported, but not the aging data.
(compare to "Export for Cluster Computations")

**Comment by msevestre** [comment]
Aging data are only created when combined in a simulation no?

**Comment by Yuri05** [comment]
yes, exactly. It's about exporting population info from a population **simulation**.
